storefront/app/controllers/spree/checkout_controller.rb

```
@@ -179,8 +187,23 @@ def load_order
           flash[:error] = 'You cannot access this checkout'
           redirect_to_cart
         elsif try_spree_current_user.nil? && !@order.completed?
-          store_location
-          redirect_to spree_login_path
+          if params[:guest] && current_store.prefers_guest_checkout?
+            @order = current_store
+                       .orders
```

RuboCop (1.73)

[convention] Place the . on the previous line, together with the method call receiver.

(Layout/DotPosition)